static/templates/items/backpack-details.hbs

```diff
@@ -17,7 +24,4 @@
     <input type="number" name="system.bulk.ignored" id="{{fieldIdPrefix}}ignored" value="{{data.bulk.ignored}}" />
 </div>
```

Comment:
You'll want to indent these form groups and remove the empty space after the if and before the end/if to maintain convention.

Reply:
Done.